gettext.opam

```diff
@@ -21,7 +21,7 @@ depends: [
   "cppo" {build}
   "fileutils"
   "base-bytes"
-  "ounit" {with-test & > "2.0.8"}
+  "ounit2" {with-test & > "2.2.7"}
```

Comment:
To make this backward compatible with `4.03.0`, one needs to link the tests with `seq` and and add it as a test-dependency here.

Reply:
Good catch, added.